tests/modules/programs/yazi/default.nix

```diff
@@ -1,5 +1,6 @@
 {
   yazi-settings = ./settings.nix;
+  yazi-init-lua-string = ./init-lua-string.nix;
```

Comment:
We should make another test `init-lua-path.nix` for when the input is a path:

```suggestion
  yazi-init-lua-path = ./init-lua-path.nix;
  yazi-init-lua-string = ./init-lua-string.nix;
```

```nix
# init-lua-string.nix
{ ... }: {
  programs.yazi = {
    enable = true;

    initLua = ./init.lua;
  };

  test.stubs.yazi = { };

  nmt.script = ''
    assertFileContent home-files/.config/yazi/init.lua \
      ${./init.lua}
  '';
}
```

Reply:
This is already covered by https://github.com/nix-community/home-manager/blob/36e2f9da91ce8b63a549a47688ae60d47c50de4b/tests/modules/programs/yazi/settings.nix#L72 though right?

Reply:
Yup, you're right. I didn't see that before.